hack/arktos-apiserver-partition.md

```diff
@@ -0,0 +1,172 @@
+
+# Arktos Apiserver Partition
+
+[![Go Report Card](https://goreportcard.com/badge/github.com/futurewei-cloud/arktos)](https://goreportcard.com/report/github.com/futurewei-cloud/arktos)
+[![LICENSE](https://img.shields.io/badge/license-apache%202.0-green)](https://github.com/futurewei-cloud/arktos/blob/master/LICENSE)
+
+
+## What Arktos Apiserver Partition is
+Arktos Apiserver Partition is a lightweight componized script to deploy Arktos by its functions. It runs like bash hack/arktos-apiserver-partition.sh function to achieve flexiblity.
+
+## Key Features of Arktos Apiserver Partiion
+
+### Partitioned 
+Feel free to export RANGE_BEGIN=aaa, RANGE_END=zzz to make apiservers only watches data you want to sync
+
+### Deploy across hosts
+Since we build a shared ETCD cluster, all the data can be shared across hosts
+
+### Build Arktos
+
+
+To build Arktos, you just need to clone the repo and run "make":
+
+##### Note: you need to have a working [Go 1.12 environment](https://golang.org/doc/install). Go 1.13 is not supported yet.
+
+```
+mkdir -p $GOPATH/src/github.com
+cd $GOPATH/src/github.com
+git clone https://github.com/futurewei-cloud/arktos
+cd arktos
+make quick-release
+
```

Comment:
From clean machine, I believe you need to create k8s.io/kubernetes symlink so arktos can build. So don't write too details about dev environment setup, otherwise, you are forced to be accurate on each single step.

Reply:
Not now. I build several times at clean aws instances. We don't need to create symlink any more.

I removed the details.

Reply:
Seems like there is a missing ``` that caused code formatting offset.

Reply:
Added